default.json

```diff
@@ -54,6 +54,20 @@
         "gomodUpdateImportPaths",
         "gomodTidy"
       ]
+    },
+    {
+      "description": "Override the docker:pinDigests preset provided by config:best-practices",
+      "matchDatasources": [
```

Comment:
```suggestion
      "matchManagers": [
```

Reply:
As I said on Slack, I used `matchDatasources` because that's what the preset use; I'm concerned that, if we use a different conditional, the settings might not be applied in the order that we hope, and we'll end up with pinning enabled.